####################
# GROUND REMOVAL
####################

# SMOOTH WINDOW
smooth_window_size_front: 5
smooth_window_size_left: 5
smooth_window_size_right: 5

# GROUND REMOVAL THRESHOLD
ground_remove_angle_front: 18
ground_remove_angle_left: 18
ground_remove_angle_right: 18





